### DISCUSSION TITLE
Does 3.0.0-M use any of the /application /app-name_profile type loading at all for secrets?

### DISCUSSION THREAD

**Discussion by mikelhamer** [awspring/spring-cloud-aws#593] [Q&A] [answered]
Comparing the docs for version 2.4.x and 3.0.0-M, it looks like 3.0.0-M no longer will automatically load common properties under /secret/application, service specific properties under /secret/service-name, with or without _profile specifications, etc. 

Does this still happen at all in 3.0.0-M or does it strictly only use the secrets specified individually in spring.config.import?

**Comment by maciejwalkowiak** [comment, marked as answer]
There was a lot of confusion about this feature and lots of edge cases where it did not work as users expected so we decided to make this feature less magical but more explicit. At this stage you need to specify exactly the secret you need.

We may re-consider but so far we haven't received feedback to do so.

**Reply by mikelhamer** [reply]
Thanks for the explanation. 

Doesn't this kind of conflict with the direction of Spring Cloud Config though? Seems their latest release is still using this pattern.
https://docs.spring.io/spring-cloud-config/docs/current/reference/html/#_aws_secrets_manager

I'm probably just misunderstanding something, or perhaps this split in direction is intentional?

**Comment by PatrickGotthard** [comment]
Personally I found this feature very useful. Now I have to declare all imports explicitly for different profiles.

**Reply by maciejwalkowiak** [reply]
I understand but got dozens of issues reported regarding this mechanism and it was just too time consuming to maintain.
